tests/Subutai.Repository.SqlRepository.Tests/ProjectEntityRepositoryTests.cs

```diff
@@ -89,4 +90,93 @@ public async Task AddAsync_ShouldHandleException()
         // Assert
         await act.Should().ThrowAsync<Exception>();
     }
+    #endregion
+
+    #region update method test
+    [Fact]
+    public async Task UpdateAsync_ShouldUpdateProjectEntity()
+
+    {
+        // Arrange
+        var existingEntity = new ProjectEntity
+        {
+            Id = 1,
+            Name = "Old Name"
+        };
+
+        _context.Projects.Add(existingEntity);
+        await _context.SaveChangesAsync();
+
+        var updateEntity = new ProjectEntity
+        {
+            Id = existingEntity.Id,
+            Name = "New Name"
+        };
+
+        // Act
+        var result = await _repository.UpdateAsync(updateEntity);
+
+        // Assert
+        result.Should().NotBeNull();
+        result.Id.Should().Be(existingEntity.Id);
+        result.Name.Should().Be(updateEntity.Name);
+    }
+     [Fact]
+    public async Task UpdateAsync_ShouldThrowArgumentException_WhenEntityNotFound()
+    {
+    // Arrange
+    var ExistentEntity = new ProjectEntity { Id = 1, Description="first Name" };
```

Comment:
[Suggestion]
camel case is preferred. also existing instead of existent

```suggestion
    var existingEntity = new ProjectEntity { Id = 1, Description="first Name" };
```